Update Process for Instating Trusted Committers (#141)
gruetter/innerSourceTest: TRUSTED-COMMITTERS.md

# Trusted Committers

Trusted committers (TCs) are those members of our working group who have elevated rights and direct write access to this repository. *TCs act as stewards of the working group and community. They aim to make consensus-based decisions in the best interest of the working group.* They also act as the guardians of this repository: TCs react to, referee, and give feedback about incoming contributions.

> ## Current Trusted Committers

Reviewer: we could also think about referring to a `.github/codeowners` file with the listed persons handles in it. 
That would this would give [a bit of automation](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners).

Reviewer 2: Just do it. :)
CODEOWNERS are awesome, and it is a cheap experiment.

Reviewer: I'll try that once this gets merged.

We follow this process (adapted from [here](https://tech.europace.de/voting-in-new-trusted-committers/)):

1. Any trusted committer (TC) can step forward and nominate a new TC in the private Slack channel #innersource-patterns-tcs. The TC should provide the following information:
   * Name of the candidate
   * Reason for candidate
   * Github handle of the candidate
   * Slack handle of the candidate
1. Every TC can raise concerns or second the nomination in the #innersource-patterns-tcs channel.
1. If none of the existing TCs disagrees with the nomination within 72h, [lazy consensus](https://tech.europace.de/lazy-consensus-vs-explicit-voting/) is reached: The nomination is accepted.
1. The TC who nominated the candidate informs her/him in private about the nomination and its acceptance. The candidate can decide on whether to accept or reject the offer.
1. If the candidate accepts the offer, the TC who nominated the candidate, makes sure:
   * New TC is added to this file
   * New TC receives write access to this repository
   * New TC is added to the #innersource-patterns-tcs channel

>    * New TC is praised on the general InnerSource Commons [mailing list](mailto:innersource-commons@googlegroups.com)

Reviewer 3: ```suggestion
   * New TC is praised in the [#innersource-patterns](https://app.slack.com/client/T04PXKRM0/C2EFRTS6A) channel.
```

I don't think that the entire commons needs to be notified - just the people that are interested in the InnerSource Patterns working group.

Author: I agree with the proposed change (and already committed it to the PR branch). 

For documentation purpose: Our idea was to give praise to a wide audience, also to recognize (and motivate) new committers. I agree though, for the beginning, it will be likely be sufficient to announce it in the patterns channel.

## Admins
A handful of individuals are currently the technical GitHub Admins for this repository. This includes most members of the InnerSource Commons' #tech-infra team and members of the [InnerSource Commons GitHub Organization](https://github.com/innersourcecommons). However, please channel working group-specific requests through the trusted committers.

> ## References

Reviewer 2: @maxcapraro do you want to keep this headline? That section doesn't have any content, so when looking at the rendered HTML it looks like there is something missing. I can send a PR to remove the section if you like.

Author: Nope. Thank you :) I deleted it.
